app/__init__.py

```diff
@@ -78,6 +78,16 @@ def post_time_line_post():
     name = request.form['name']
     email = request.form['email']
     content = request.form['content']
+    if not name or not name.strip():
```

Comment:
@aliaaabdulahi I'm not sure why this response doesn't pass the tests for bad requests